File: Consultora/FrontEnd/src/app/models/ClientInfo.ts
export class ClientInfo {

    name: string;
    numberOfVariables: number;
    numberOfObjectives: number;
    lowerLimit: number[];
    upperLimit: number[];

    constructor(name: string,
        numberOfVariables: number,
        numberOfObjectives: number,
        lowerLimit: number[],
        upperLimit: number[]) { 
            this.name = name;
            this.numberOfVariables = numberOfVariables;
            this.numberOfObjectives = numberOfObjectives;
            this.lowerLimit = lowerLimit;
            this.upperLimit = upperLimit;
    }
}

File: Consultora/Server/src/main/java/com/example/sever/models/ClientInfo.java
package com.example.sever.models;

import org.uma.jmetal.problem.impl.AbstractIntegerProblem;
import org.uma.jmetal.solution.IntegerSolution;

import java.io.Serializable;

public class ClientInfo extends AbstractIntegerProblem implements Serializable {

    private static final long serialVersionUID = 1L;

    public ClientInfo(ClientInfoInput clientInfoInput) {
        super();
        setName(clientInfoInput.name);
        setNumberOfVariables(clientInfoInput.numberOfVariables);
        setNumberOfObjectives(clientInfoInput.numberOfObjectives);
        setLowerLimit(clientInfoInput.lowerLimit);
        setUpperLimit(clientInfoInput.upperLimit);
    }

    @Override
    public void evaluate(IntegerSolution integerSolution) {

    }
}
